vm/instruction.go

```diff
@@ -453,6 +453,8 @@ func (vm *VM) initObjectFromGoType(value interface{}) Object {
 	switch v := value.(type) {
 	case int:
 		return vm.initIntegerObject(int(v))
+	case int64:
```

Comment:
Yeah, and it is not a good practice to convert `int64` back to `int` type which might cause value being truncated.